### DISCUSSION TITLE
How to extend the ceph filesystem if we reach the maximum size of OSD disks ?

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#7369] [Q&A]
I'm running a rook-ceph cluster with 3 OSDs which span 3 EBSs using custer-on-pvc.yaml , everything is running as expected but we know that the maximum size of those EBSs is 16TBs, we presume that in the future we'll hit that limit, in this case, all OSDs will be filled up, we think of using more OSDs with grouped replication but i don't know if that possible.
 If i add additional OSDs that won't make a difference since all data is replicated across them or does using more OSDs than the actual Replicated Data Pool work??

**Comment by travisn** [comment]
Let's discuss on https://github.com/rook/rook/discussions/7367
